app/admin/regional_holiday.rb

```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
 
 ActiveAdmin.register RegionalHoliday do
-  permit_params :name, :day, :month, office_ids: []
+  permit_params :name, :day, :month, office_ids: [], city_ids: []
 
   filter :name
   filter :offices, multiple: true, collection: -> { Office.active.order(:city) }
+  filter :cities, multiple: true, collection: -> { City.all.order(:name) }
```

Comment:
you can do `City.order(:name)`

Reply:
I didn't know it was possible to do it that way. It's fixed now!